server/database/sequelize.ts

```diff
@@ -32,3 +34,27 @@ export const sequelize = new Sequelize(url, {
     idle: 10000,
   },
 });
+
+export const umzug = new Umzug({
```

Comment:
```suggestion
export const migrations = new Umzug({
```

Lets export this as "migrations"